### PR TITLE
Fixes #3329 - Fix duplicate CSS in production builds

### LINKED ISSUE #3329: Duplicate CSS
- swagger-ui version
3.x
- a swagger file reproducing the issue
http://petstore.swagger.io/

I'm in the process of updating Swashbuckle with swagger-ui 3.x an I noticed that the CSS for the ```input.download-url-input``` shows duplicate style in the developer tools, it seems one is from a <style> in the body and the other from the .css file.

image

### PR DESCRIPTION
**Pull request by owenconti**
Fixes #3329 

Ping @shockey and @Minasokoni 

- Made a re-usable style config file for production build style loaders
- Added ExtractTextPlugin plugin to the production build style config file
- Changed `separateStylesheets` to `true` in all of the dist config files
- My editor is configured to automatically fix files based on eslint rules, so it fixed all the single quotes to double quotes, and removed semi-colons

### CODE REVIEW

File: make-webpack-config.js [hunk 1]
```diff
@@ -54,13 +54,12 @@ module.exports = function(rules, options) {
 
   if( specialOptions.separateStylesheets ) {
     plugins.push(new ExtractTextPlugin({
-      filename: '[name].css' + (specialOptions.longTermCaching ? '?[contenthash]' : ''),
+      filename: "[name].css" + (specialOptions.longTermCaching ? "?[contenthash]" : ""),
```
Comment (owenconti) [inline make-webpack-config.js]: One "downside" to my changes: `swagger-ui-bundle.css` and `swagger-ui-standalone-preset.css` files are now generated with the build. We could "fix" that by changing the `[name].css` to `swagger-ui.css`.

Comment (shockey) [inline make-webpack-config.js]: @owenconti, I'd like for that to be fixed before merging - in my mind, the dist folder contents are part of our public API surface, so we should avoid changing the file names if at all possible.

Comment (shockey) [inline make-webpack-config.js]: @owenconti and I had an out-of-band conversation, and settled on hardcoding `swagger-ui.css`.

**Comment by owenconti** [comment]
@shockey I pushed it a bit further than we discussed. Since all of the styles are built into `swagger-ui.css`, I updated the bundle and standalone builds to not compile any styles. This now results in the original, single `.css` file. 

I also noticed that there was only one `.less` file in the project, so I moved that into a `.scss` file and then removed the `less-loader` webpack config and dependency.

**Pull request merged**